### PR TITLE
Changes According to Fuze API

### PR DESCRIPTION
**Pull request by KMayuri25**

### CODE REVIEW

File: src/app/partner/click-partner-row/click-partner-row.component.ts [hunk 1]
```diff
@@ -27,7 +27,12 @@ export class ClickPartnerRowComponent implements OnInit {
     this.rowData = params.data;
   }
   onClick() {
+console.log(this.rowData);
```
Comment (saiprasad1997) [inline src/app/partner/click-partner-row/click-partner-row.component.ts]: remove logging statement

File: src/app/partner/update-partner-fuze/update-partner-fuze.component.ts [hunk 2]
```diff
@@ -107,6 +108,25 @@ export class UpdatePartnerFuzeComponent implements OnInit {
     //   });
     //
     // }
+
+    if (this.getSwitchValue()) {
+      this.partnerObject.status = 'inActive';
+        this.partnerService.update_fuse(this.id, this.partnerObject).subscribe((response: Response) => {
+          this.router.navigate(['partner']);
+        },error => {
+          localStorage.setItem('ErrorMsg', error );
```
Comment (saiprasad1997) [inline src/app/partner/update-partner-fuze/update-partner-fuze.component.ts]: why error message is logged in localstorage

File: src/app/partner/update-partner-fuze/update-partner-fuze.component.ts [hunk 3]
```diff
@@ -107,6 +108,25 @@ export class UpdatePartnerFuzeComponent implements OnInit {
...
+          this.showErrorMsg = true;
+        });
+    } else {
+
+      this.partnerService.update_fuse(this.id, this.partnerObject).subscribe((response: Response) => {
+        this.router.navigate(['partner']);
+      },error => {
+        localStorage.setItem('ErrorMsg', error );
```
Comment (saiprasad1997) [inline src/app/partner/update-partner-fuze/update-partner-fuze.component.ts]: remove error message logged in local storage

File: src/app/partner/create-partner-fuse/create-partner-fuse.component.html [hunk 4]
```diff
@@ -101,12 +110,19 @@ <h1 class="page-title">Register New Partner</h1>
...
-            <div class="left-textbox pull-left">
-              <label class="lbl-size  " translate="tenantUsersTab.firstName">MEDUSA ENDPOINT </label>
+            <!--<div class="left-textbox pull-left">-->
+              <!--<label class="lbl-size  " translate="tenantUsersTab.firstName">MEDUSA ENDPOINT </label>-->
+              <!--<span class="required-star">*</span>-->
+              <!--<div class="item">-->
+                <!--<input id = 'medusa_endpoint' type="text" class="form-control" placeholder="Medusa Endpoint" formControlName="medusa_endpoint" />-->
+
+              <!--</div>-->
+            <!--</div>-->
```
Comment (pooja-koul-incontact) [inline src/app/partner/create-partner-fuse/create-partner-fuse.component.html]: remove commented code

File: src/app/partner/create-partner-fuse/create-partner-fuse.component.html [hunk 5]
```diff
@@ -127,22 +143,16 @@ <h1 class="page-title">Register New Partner</h1>
...
-              <div class="item">
-                <input id = 'app_token' type="text" class="form-control" placeholder="App Token" formControlName="app_token"  />
-              </div>
-            </div>
+            <!--<div class="left-textbox pull-left">-->
+              <!--<label class="lbl-size  " translate="tenantUsersTab.firstName">APP TOKEN </label>-->
+              <!--<span class="required-star">*</span>-->
+              <!--<div class="item">-->
+                <!--<input id = 'app_token' type="text" class="form-control" placeholder="App Token" formControlName="app_token"  />-->
+              <!--</div>-->
```
Comment (pooja-koul-incontact) [inline src/app/partner/create-partner-fuse/create-partner-fuse.component.html]: remove commented code

File: src/app/partner/update-partner-fuze/update-partner-fuze.component.ts [hunk 6]
```diff
@@ -79,14 +80,14 @@ export class UpdatePartnerFuzeComponent implements OnInit {
...
     this.partnerObject.username = this.control.username.value;
     this.partnerObject.password = this.control.password.value;
-    this.partnerObject.config = (JSON.stringify(this.config_val = {'tenantKey': this.control.tenantKey.value, 'medusa_endpoint': this.control.medusa_endpoint.value, 'warden_authURI': this.control.warden_authURI.value, 'app_token': this.control.app_token.value, 'presenceEndpoint': this.control.presenceEndpoint.value, 'bossAPI': this.control.bossAPI.value}));
-  //  this.partnerObject.email = this.control.email.value;
+    this.partnerObject.config = (JSON.stringify(this.config_val = { 'wardenAPIURI': this.control.warden_authURI.value, 'appToken': this.control.app_token.value, 'bossAPIURI': this.control.bossAPI.value}));
+    this.partnerObject.email = this.control.email.value;
     this.partnerObject.status = 'active';
 
-
+console.log(this.partnerObject);
```
Comment (pooja-koul-incontact) [inline src/app/partner/update-partner-fuze/update-partner-fuze.component.ts]: remove logger

**Pull request merged**